profiles/car.lua

```diff
@@ -11,6 +11,9 @@ access_tags_hierarchy = { "motorcar", "motor_vehicle", "vehicle", "access" }
 service_tag_restricted = { ["parking_aisle"] = true }
 restriction_exception_tags = { "motorcar", "motor_vehicle", "vehicle" }
 
+-- A list of suffixes to suppress in name change instructions
+suffix_list = { "N", "NE", "E", "SE", "S", "SW", "W", "NW", "road", "street" }
```

Comment:
If you feel these are significant, I can remove road/street. I could easily do a suffix check as well and do them exclusively, so not prefix + suffix. That would enable north/south changes but not 
North Bend Road to Bend.